### ISSUE TITLE
data_colors fails when there are less than 3 groups

### ISSUE THREAD

**Issue by aviezerl** [tanaylab/chameleon#1]
Example: 

```r
data <- data.frame(umap_x = c(17.0890121459961, 13.7146520614624, 
16.1004028320312, 11.98228931427, 14.5449142456055, 16.459436416626, 
15.6826705932617, 16.2683963775635, 9.76927852630615, 10.5458135604858
), umap_y = c(1.55449628829956, 6.1767144203186, -0.520148098468781, 
10.9073801040649, 9.17697143554688, 8.81098747253418, -0.728159248828888, 
7.35069417953491, 10.131459236145, 9.69721508026123), umap_u = c(11.1400880813599, 
12.1072673797607, 10.1809425354004, 9.63253021240234, 9.83402347564697, 
9.0894193649292, 9.80254173278809, 10.2694730758667, 10.9983005523682, 
10.9255685806274))

groups <- rep("savta", nrow(data))
chameleon::data_colors(data, groups = groups, run_umap = FALSE)
#> Error in normalized_data(stats::prcomp(data, retx = TRUE)$x): ncol(data) == 3 is not TRUE
```

When there are 3 groups, everything works perfectly: 

```r
groups <- sample(c("savta", "saba", "doda"), size = nrow(data), replace = TRUE)
chameleon::data_colors(data, groups = groups, run_umap = FALSE)
#> doda'#E0610F'saba'#BE68BC'savta'#4D976A'
```

**Comment by orenbenkiki** [comment]
Pushed a fixed version which you can install from github.

It is in the process of being pushed to CRAN as well (as 0.2.1).

**Comment by orenbenkiki** [comment]
Hmmm... not quite there. Will fix.

**Comment by orenbenkiki** [comment]
Fixed in 0.2.2 - install latest from CRAN and all should be well.
